RELEASE/scripts/autoscend/auto_island_war.ash

```diff
@@ -256,12 +303,11 @@ boolean L12_startWar()
 		return false;
 	}
 
-	auto_log_info("Must save the ferret!!", "blue");
-	warOutfit(false);
 	if((my_mp() > 60) || considerGrimstoneGolem(true))
 	{
 		handleBjornify($familiar[Grimstone Golem]);
 	}
+	
 	buffMaintain($effect[Snow Shoes], 0, 1, 1);
 	buffMaintain($effect[Become Superficially Interested], 0, 1, 1);

```

Comment:
remove these. the provider call in the next line should be handling this.

Reply:
change made